src/WeekFour/HomeworkSets.java

```diff
@@ -0,0 +1,77 @@
+package WeekFour;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class HomeworkSets {
+    public static void main(String[] args){
+        /*
+        *
+        * Sets:
+          Lesson 11 – Homework – Sets
+          Initiate a Set. Using For loop add integers from 1 to 10
+          Remove all odd numbers from sets.
+          Try to create a set with duplicate values
+        * */
+        System.out.println("\n#1");
+        loopSet(10);
+        System.out.println("\n#2");
+        removeOddSet(15);
+        System.out.println("\n#3");
+        duplicateValues();
+
+    }
+
+    public static void loopSet(int num){
+        int number = num;
+
+        Set<Integer> loop_num = new HashSet<>();
+
+        for(int i = 1; i <= number; i++ ){
+            loop_num.add(i);
+        }
+
+        for (int set : loop_num){
+            System.out.println("Counting: " + set);
+        }
+    }
+
+    public static void removeOddSet(int num){
```

Comment:
This is a good solution but you have to initialize and add numbers to the set and then remove all odd numbers from the set using "remove" method

Reply:
@vladimirKorzhukov 
Please recheck when you have the chance.